CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ All versions prior to 0.9.0 are untracked.
 * API: Make Rekor APIs compatible with Rekor v2 by removing trailing slashes
   from endpoints ([#1366](https://github.com/sigstore/sigstore-python/pull/1366))
 
+* Timestamps: Verify the signature date against the validity period of both the
+  Timestamp Authority or the Transperency Service, if either of such timestamps
+  are present in the Bundle. We still require at lease one of such timestamps.
```

Comment:
Let's not call log integration time a timestamp. Also _"validity period of Timestamp Authority"_ does not sound correct -- In the code you modify we're comparing the established times (timestamps or log integration time) to the validity period of the signing certificate.

Maybe:
```suggestion
* Verify: verify that all established times (timestamps or the log integration time)
  are within the signing certificate validity period. At least one established time is
  still required.
```